.github/workflows/release.yaml

```diff
@@ -0,0 +1,22 @@
+name: Create release packages
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  package:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - name: Build releases
+        run: make builds
+      - name: Creating release tarballs
+        run: make archives
+      - name: Creating release packages
+        run: make packages
```

Comment:
`make packages` already builds the archives too so we can remove the archive step